packages/crypto/src/networks/devnet/milestones.json

```diff
@@ -21,9 +21,11 @@
                 "ipfs": 0,
                 "timelockTransfer": 0,
                 "multiPayment": 0,
-                "delegateResignation": 2500000000
+                "delegateResignation": 2500000000,
+                "businessRegistration": 5000000000
             }
         },
+        "aip11": true,
```

Comment:
Revert this please. That's only for `testnet` at the moment until we move to devnet.